### PR TITLE
feat: added the FormAssesment and Question APIs

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/assesment/views.py [hunk 1]
```diff
@@ -51,5 +53,67 @@ def perform_destroy(self,serializer):
             appointment = serializer.save()
 
 
+
+
+class QuestionViewSet(viewsets.ViewSet):
```
Comment (Raam124) [inline apps/assesment/views.py:58]: this only have one GET API why are you using a viewset here ?

File: apps/assesment/views.py [hunk 2]
```diff
@@ -51,5 +53,67 @@ def perform_destroy(self,serializer):
...
+        if treatment_type:
+            questions = Question.objects.filter(treatment_type__treatment_option=treatment_type)
+        else:
+            questions = Question.objects.all()
+        serializer = QuestionSerializer(questions, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+
+class FormAssesmentViewSet(viewsets.ModelViewSet):
```
Comment (Raam124) [inline apps/assesment/views.py:74]: check the features of form assessment feature once again

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/assesment/views.py:64]: Rechecked and made changes

File: apps/assesment/views.py [hunk 3]
```diff
@@ -50,6 +53,67 @@ def perform_destroy(self,serializer):
...
+    serializer_class = QuestionSerializer
+    permission_classes = [permissions.IsAuthenticated,]
+
+
+class FormAssesmentViewSet(viewsets.ModelViewSet):
+    queryset = FormAssesment.objects.all()
+    serializer_class = FormAssesmentSerializer
+    permission_classes = [permissions.IsAuthenticated,]
+
+    def perform_create(self, serializer):
```
Comment (Raam124) [inline apps/assesment/views.py:69]: what are the data that you getting here for form assessment post ?
how can you only get the answers without the questions ?

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 3 above)
Comment (Kavindu-248) [reply apps/assesment/views.py:67]: resolved to get the questions by filtering with the treatment type.

**Review by Raam124 — APPROVED** [review]

**Pull request merged**